.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "autovot"]
```

Comment:
Am I right in that this is only here for testing purposes to get data for testing?  If so, I'd prefer to have our own test data or just copy over a file or two of theirs (with permission), rather than have the full autovot repo be a submodule.

Reply:
Yeah that is the only reason, and I agree it's kinda clunky. I can probably just switch it over to a different file and use the SOTC classifier that's already included in PolyglotDB anyhow.

Reply:
Ok, yeah, let's do that.  My thinking is we just need to run autovot on some arbitrary file, get the output there and do some checks that when conch wraps around it it's returning the correct information.  I.e., all we need to do is test the conch code, we don't really care about autovot's accuracy per se.